packages/v/vid.stab/abi_used_symbols

```diff
@@ -9,7 +9,10 @@ libc.so.6:fgetc
 libc.so.6:fgets
 libc.so.6:fopen
```

Comment:
Have you re-built this package since we updated the toolchain (glibc/gcc/binutils/etc)? I think this ABI report indicates that you haven't. Can you please re-build it against the latest stable?

Reply:
So I tried to rebuild the package from my local clone of vid.stab branch, there is no change of abi_used_symbols. I even tried to update it from main branch and build it. The file output still the same. I already did `git pull` before doing any changes on main branch.